saleor/static/dashboard-next/siteSettings/components/SiteSettingsDetails/SiteSettingsDetails.tsx

```diff
@@ -0,0 +1,92 @@
+import Card from "@material-ui/core/Card";
+import CardContent from "@material-ui/core/CardContent";
+import TextField from "@material-ui/core/TextField";
+import * as React from "react";
+
+import CardTitle from "../../../components/CardTitle";
+import FormSpacer from "../../../components/FormSpacer";
+import i18n from "../../../i18n";
+import { SiteSettingsPageFormData } from "../SiteSettingsPage";
+
+interface SiteSettingsDetailsProps {
+  data: SiteSettingsPageFormData;
+  errors: Partial<{
+    description: string;
+    domain: string;
+    name: string;
+  }>;
+  disabled: boolean;
+  onChange: (event: React.ChangeEvent<any>) => void;
+}
+
+const SiteSettingsDetails: React.StatelessComponent<
+  SiteSettingsDetailsProps
+> = ({ data, disabled, errors, onChange }) => (
+  <Card>
+    <CardTitle
+      title={i18n.t("General Information", {
+        context: "card title"
```

Comment:
Better context: "store configuration". Physical placement (if important) could be part of a translator comment (not possible until we switch to a different translation API).